src/UI/Art/SpriteArtist/InfoView.java

```diff
@@ -77,22 +78,47 @@ public void render(Graphics g)
     switch (page.pageType)
     {
       case CO_HEADERS:
-        int overlayHeight = 42;
+        int coEntryHeight = 42;
         int heightOffset = 0;
-        for (Commander CO : myControl.getGame().commanders)
+
+        Commander[] coList = myControl.getGame().commanders;
+        int coIndex = myControl.getGame().getCOIndex(thisCO);
+
+        int selectedOptionYOffset = coIndex * coEntryHeight;
+        int verticalShift = 0; // How many COs to skip drawing "off the top"
+        int displayableCount = paneVSize / coEntryHeight; // how many COs we can cram on the screen
+        while (selectedOptionYOffset > paneVSize / 2 && // Loop until either the cursor's bumped up to the center of the screen...
+            displayableCount + verticalShift < coList.length) //  or we'll already show the last CO 
+        {
+          verticalShift++;
+          selectedOptionYOffset -= coEntryHeight;
+        }
+
+        for( int i = verticalShift; i < coList.length; ++i )
         {
+          Commander CO = coList[i];
+
           // Build our overlay
-          BufferedImage overlayPic = SpriteLibrary.createTransparentSprite(drawingWidth, overlayHeight);
+          BufferedImage overlayPic = SpriteLibrary.createTransparentSprite(drawingWidth, coEntryHeight);
+          Graphics overlayG = overlayPic.getGraphics();
+
+          // Highlight the selected CO
+          if( i == coIndex )
+          {
+            overlayG.setColor(new Color(246, 234, 210));
```

Comment:
Why not `SpriteUIUtils.MENUHIGHLIGHTCOLOR`?

Reply:
Forgot it existed.